database/seeds/ArticlesTableSeeder.php

```diff
@@ -69,7 +69,140 @@ public function run()
                 ],
             ],
             [
-                'title' => 'Grand spectacle du PAE',
+                'title' => 'Grand spectacle du PAE 1',
```

Comment:
I suggest we generate articles with [Faker](https://github.com/fzaninotto/Faker). If you want to see it in action, I did it once [here](https://gitlab.utc.fr/compiegne-en-transition/site-web/compiegne_en_transition/blob/master/database/seeds/ArticleTableSeeder.php). It will help #303 resolution.

Reply:
Finally, we decided with @aopom adandon the article seeder modifications. #303 resolution will improve Articles seeder.